internal/cluster/cluster.go

```diff
@@ -266,9 +266,12 @@ func (m *Manager) reportValidationStatusChanged(ctx context.Context, c *common.C
 				if v.Status == ValidationFailure && currentStatus == ValidationSuccess {
 					m.metricAPI.ClusterValidationChanged(models.ClusterValidationID(v.ID))
 					eventgen.SendClusterValidationFailedEvent(ctx, m.eventsHandler, *c.ID, v.ID.String(), v.Message)
-				}
-				if v.Status == ValidationSuccess && currentStatus == ValidationFailure {
+				} else if v.Status == ValidationSuccess && currentStatus == ValidationFailure {
```

Comment:
Same as [this](https://github.com/openshift/assisted-service/pull/4196/files#r932467611) but for clusters you don't have supress and you also don't have disabled